docs/reference/runtime-fields.asciidoc

```diff
@@ -1,15 +1,17 @@
-[[alerts-runtime-fields]]
-== Add runtime fields to detection alerts
+[[runtime-fields]]
+== Create runtime fields in {elastic-sec}
 
 Runtime fields are fields that you can add to documents after you've ingested your data. For example, you could combine two fields and treat them as one, or perform calculations on existing data and use the result as a separate field. Runtime fields are evaluated when a query is run. 
 
-You can add a runtime field to your detection alerts from the *Alerts* table. The new field applies to all {elastic-sec} alerts.
+You can create a runtime field and add it to your detection alerts or events from any page that lists alerts or events in a data grid table, such as *Alerts*, *Timelines*, *Hosts*, and *Users*. Once created, the new field becomes available to all {elastic-sec} alerts and events.
```

Comment:
I would maybe mention that the field gets added to the Security Solution data view and is applied to all alerts and events in the data view

Reply:
@stephmilovic Thanks, revised! I specified "added to the current data view" because in testing this out, I saw the new field gets added to whatever data view is currently selected on that page. In most cases this is the default Security data view (such as on the Alerts page where the user can't change the data view), but on other pages like Timeline it's possible to select a different data view. Also added a link to more detailed docs on data views in the Security app for more context.